frontend/dockerfile/linter/ruleset.go

```diff
@@ -0,0 +1,110 @@
+package linter
+
+import (
+	"strings"
+
+	"github.com/moby/buildkit/frontend/dockerfile/parser"
+)
+
+const (
```

Comment:
Make sure we add the existing line-continuation parser warning also to this list.